db/change_cache.go

```diff
@@ -12,16 +12,16 @@ import (
 	"sync"
 	"time"
 
-	"github.com/couchbase/sg-bucket"
+	sgbucket "github.com/couchbase/sg-bucket"
 	"github.com/couchbase/sync_gateway/base"
 	"github.com/couchbase/sync_gateway/channels"
 )
 
 const (
-	DefaultCachePendingSeqMaxNum  = 10000            // Max number of waiting sequences
-	DefaultCachePendingSeqMaxWait = 5 * time.Second  // Max time we'll wait for a pending sequence before sending to missed queue
-	DefaultSkippedSeqMaxWait      = 60 * time.Minute // Max time we'll wait for an entry in the missing before purging
-	QueryTombstoneBatch           = 250              // Max number of tombstones checked per query during Compact
+	DefaultCachePendingSeqMaxNum  = 10000           // Max number of waiting sequences
+	DefaultCachePendingSeqMaxWait = 5 * time.Second // Max time we'll wait for a pending sequence before sending to missed queue
+	DefaultSkippedSeqMaxWait      = 2 * time.Minute // Max time we'll wait for an entry in the missing before purging
```

Comment:
(Commenting in the correct place this time...)

Accidental commit?

Reply:
Yup, was for testing! Thanks